doc/modules/impute.rst

```diff
@@ -105,7 +105,16 @@ of ``y``.  This is done for each feature in an iterative fashion, and then is
 repeated for ``max_iter`` imputation rounds. The results of the final
 imputation round are returned.
 
+.. note::
```

Comment:
Could you make a small change in l. 25 and 27 of this file. The hyperlinks does not work:

* :class:`impute.SimpleImputer` -> :class:`SimpleImputer`
* :class:`impute.IterativeImputer` -> :class:`IterativeImputer`